api/integrations/github/tasks.py

```diff
@@ -0,0 +1,89 @@
+import logging

```

Comment:
We need to add tests for this module

Reply:
I think this will be included in the hooks test above.

Reply:
All this function are tested in the `create feature external resources test`